src/providers/fakerest/dataProvider.ts

```diff
@@ -20,10 +20,11 @@ import { getActivityLog } from '../commons/activity';
 import { getCompanyAvatar } from '../commons/getCompanyAvatar';
 import { getContactAvatar } from '../commons/getContactAvatar';
 import { CrmDataProvider } from '../types';
+import { authProvider, USER_STORAGE_KEY } from './authProvider';
 import generateData from './dataGenerator';
 import { withSupabaseFilterAdapter } from './internal/supabaseAdapter';
 
-const baseDataProvider = fakeRestDataProvider(generateData(), true, 300);
+const baseDataProvider = fakeRestDataProvider(generateData(), false, 300);
```

Comment:
question: why that change?

Reply:
Too much logs in the console, debug is awful with this feature to true